internal/config/config.go

```diff
@@ -80,12 +93,15 @@ type Config struct {
 	DSN     pg.Dsn        `yaml:"dsn"`
 }
 
+// Name returns the name of this testconfig
 func (c Config) Name() (name string) {
 	return fmt.Sprintf("%s (%d)", c.path, c.index)
 }
 
-func NewConfigsFromReader(reader io.Reader, name string) (configs Configs, err error) {
+// NewConfigFromReader
+func newConfigsFromReader(reader io.Reader, name string) (configs Configs, err error) {
 	var i int
+	const zero = 0
```

Comment:
You can leave this in, but it is a bit silly.
We should check your config cause revive config allowInts: 0,1,2,3 should be ok with 0 int as a value.